linux-anvil-cuda/Dockerfile

```diff
@@ -11,8 +11,15 @@ ENV CUDA_VER ${CUDA_VER}
 # Set an encoding to make things work smoothly.
 ENV LANG en_US.UTF-8
 
-# Set path to CUDA install.
+# Set path to CUDA install (this is a symlink to /usr/local/cuda-${CUDA_VER})
 ENV CUDA_HOME /usr/local/cuda
+# the upstream images for 10.x all have libcuda.so under $CUDA_HOME/compat-$CUDA_VER;
+# add this to the ldconfig so it will be found correctly. For 9.2, the image
+# nvidia/cuda:9.2-devel-centos6 contains neither $CUDA_HOME/compat-$CUDA_VER,
+# nor any (non-stub) libcuda.so. For licensing reasons, these cannot be part of
+# the conda-forge docker images, but are instead added for CI purposes in:
+# github.com/conda-forge/conda-forge-ci-setup-feedstock/blob/master/recipe/run_conda_forge_build_setup_linux
+RUN echo "$CUDA_HOME/compat-$CUDA_VER" >> /etc/ld.so.conf.d/cuda-$CUDA_VER.conf
```

Comment:
You have to run `ldconfig` as far  as I know.

Reply:
You mean in addition to writing something to `/etc/ld.so.conf.d/cuda-9.2.conf`?
With `echo /etc/ld.so.conf` you get 
```
[root@05a32ba12702 /]# cat /etc/ld.so.conf
include ld.so.conf.d/*.conf
```
which sounds like it will be included anyway (although I haven't figured out how to check this)